src/features/Configuration.feature

```diff
@@ -0,0 +1,74 @@
+
+Feature: Configuration
+
+    Background: Required Setup
+        Given user logs in as "sysad"
+        And user selects "System Administration" on program-selection page
+
+    @BI-1502
+    Scenario Outline: Configuration as a sub-menu
+        When user is on the program-management page
+        #Create a new program
+        When user selects 'New Program' button in Programs page
+        When user sets "<ProgramName>" in Program Name field in Programs page
+        When user selects "<Species>" in Species dropdown in Programs page
+        When user sets "<Key>" in Program Key field in Programs page
+        When user selects 'Save' button in Programs page
+        When user pause for "10" seconds
+        When user navigates to Program Selection page
+        When user selects "<ProgramName>" on program-selection page
+        When user selects "Program Management" in navigation
+        When user selects "Users" tab
+        When user clicks 'New User' button
+        When user sets "Cucumber Breeder" in Name field of User
+        When user sets "cucumberbreeder@mailinator.com" in Email field of User
+        When user sets "breeder" in Role dropdown of User
+        When user click 'Save' button in User
+        When user pause for "10" seconds
```

Comment:
Is there a different way to do this that doesn't rely on time passing?

Reply:
Done.